README.md

```diff
@@ -1,4 +1,6 @@
 # Indexers
-This Repo contains Cardigann indexer definitions for [Prowlarr](https://github.com/Prowlarr/Prowlarr).
+This Repo contains Cardigann YML indexer definitions for [Prowlarr](https://github.com/Prowlarr/Prowlarr).
+
+For more information on the formatting of the YML Indexer Definition, please see [Jackett's wiki entry](https://github.com/Jackett/Jackett/wiki/Definition-format)
```

Comment:
I’d rather have our own, for when we diverge

Reply:
do we want to just grab what they have for now; toss it on the Prowlarr wiki; and call it a day?

Reply:
done. 

- wiki page created - https://wiki.servarr.com/prowlarr/cardigann-yml-definition
   - based on Jackett's page. Formatting cleaned up and some other changes.  Credit given to jackett and garfield.
- this PR updated
- prowlarr main wiki page updated
- prowlarr development updated